lib/driverProviders/useExistingWebDriver.ts

```diff
@@ -0,0 +1,57 @@
+/*
+ *  This is an implementation of the Use Existing WebDriver Driver Provider.
+ *  It is responsible for setting up the account object, tearing it down, and
+ *  setting up the driver correctly.
+ */
+import * as q from 'q';
```

Comment:
Sorry for a simple question, but why you can not use `async ... await` construction instead of `q` library?

Reply:
I mostly copied the code from another driver provider, but I could redo my PR with `async/await` if that helps it getting merged. I don't exactly know what the house rules are for protractor code but I didn't see anything prohibiting `q`, fwiw.

Reply:
The driver providers were written before Promise was available in Node.js.
Probably it's fine to use async/await for new driver provider, but I am not sure whether it will just work since some other folks tried to refactor them using regular promise, but it turns out not easy. Since all the other driver providers are using q promise, I am also ok with q promise for this one.